### PR TITLE
Revenue LTV updates

### PR DESCRIPTION
**Pull request by fbertsch**

### CODE REVIEW

File: sql/revenue_derived/client_ltv_v1/query.sql [hunk 1]
```diff
@@ -209,65 +206,105 @@ with_caps AS (
...
+    pred_num_days_searching * searches_per_day_capped * sap_value AS ltv_search_future,
+    pred_num_days_tagged_searching * tagged_searches_per_day_capped * tagged_sap_value AS ltv_tagged_search_future,
   FROM
     with_caps
+  JOIN
+    future_values
+  USING
+    (engine, country)
+),
+totals AS (
```
Comment (fbertsch) [inline sql/revenue_derived/client_ltv_v1/query.sql:267]: This was done for performance reasons

File: sql/revenue_derived/client_ltv_v1/query.sql [hunk 2]
```diff
@@ -209,65 +206,105 @@ with_caps AS (
...
-    SQRT(
-      pred_num_days_seeing_ads * searches_with_ads_per_day_capped * avg_client_search_with_ads_value
-    ) AS ltv_search_with_ads_future,
-    SQRT(
-      pred_num_days_searching * searches_per_day_capped * avg_client_search_value
-    ) AS ltv_search_future,
-    SQRT(
-      pred_num_days_tagged_searching * tagged_searches_per_day_capped * avg_client_tagged_search_value
-    ) AS ltv_tagged_search_future,
+    ad_click_days * ad_clicks_per_day_capped * ad_click_value AS ltv_ad_clicks_current,
```
Comment (fbertsch) [inline sql/revenue_derived/client_ltv_v1/query.sql:252]: Square rooting is gone!

File: sql/revenue_derived/client_ltv_v1/query.sql [hunk 3]
```diff
@@ -209,65 +206,105 @@ with_caps AS (
...
+    country
+),
+future_values AS (
+  -- For each engine-country, this gives the expected future value of each action
+  SELECT
+    engine,
+    country,
+    SUM(revenue) AS revenue,
+    SUM(ad_clicks) AS total_ad_clicks,
+    SUM(ad_click_value * (ad_clicks / total_ad_clicks)) AS ad_click_value,
```
Comment (fbertsch) [inline sql/revenue_derived/client_ltv_v1/query.sql:233]: Here's the algo we talked about (weighting value by number of ad clicks that month)

**Review by benmiroglio — APPROVED** [review]
LGTM, thanks frank

**Pull request merged**